Makefile

```diff
@@ -50,6 +53,11 @@ release-ios: prod-build-ios ##@build build release for iOS release
 prod-build:
 	lein prod-build
 
+full-prod-build: ##@build build prod for both Android and iOS
+	./scripts/bundle-status-go.sh ios
+	./scripts/bundle-status-go.sh android
+	make prod-build
```

Comment:
Could we use the $(MAKE) variable here instead?
https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

Reply:
Of course, change pushed